AFHTTPClientLogger.m

```diff
@@ -26,12 +26,29 @@
 
 #import <objc/runtime.h>
 
+
+#if __has_include(<CocoaLumberjack/DDLog.h>)
+#import <CocoaLumberjack/DDLog.h>
+// Global log level for the whole library, not per-file.
+const int ddLogLevel = LOG_LEVEL_VERBOSE;
```

Comment:
`ddLogLevel` seems to be represented as an `int` in CocoaLumberjack so I believe this is fine.